packages/graphql-transformers-e2e-tests/resources/jsonServer/cdk.context.json

```diff
@@ -1,4 +1,3 @@
 {
-  "@aws-cdk/core:enableStackNameDuplicates": "true",
```

Comment:
When I briefly debugged http transfomer e2e tests cdk v2 CLI (`aws-cdk` v2) failed fast saying that this flag is removed and enabled by default now.